exports/request_resources.go

```diff
@@ -14,13 +14,13 @@ import (
 	"github.com/redhatinsights/export-service-go/models"
 )
 
-type RequestApplicationResources func(ctx context.Context, identity string, payload models.ExportPayload)
+type RequestApplicationResources func(ctx context.Context, identity string, payload models.ExportPayload, db models.DBInterface)
 
 func KafkaRequestApplicationResources(kafkaChan chan *kafka.Message, log *zap.SugaredLogger) RequestApplicationResources {
 	// sendPayload converts the individual sources of a payload into
 	// kafka messages which are then sent to the producer through the
 	// `messagesChan`
-	return func(ctx context.Context, identity string, payload models.ExportPayload) {
+	return func(ctx context.Context, identity string, payload models.ExportPayload, db models.DBInterface) { // TODO: Does it make sense to pass the DB here?
```

Comment:
Personally, I would not pass the `models.DBInterface` instance into this method unless its needed.  If we decide we need it later on, then we can add it to the function call then.  Just my thoughts/opinions.

Reply:
Got it!